Microsoft.DotNet.Interactive.PowerShell/PowerShellExtensions.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections;
+using System.Collections.ObjectModel;
+
+namespace Microsoft.DotNet.Interactive.PowerShell
+{
+    using System.Management.Automation;
```

Comment:
Nit: You can move the using outside the namespace declaration.

Reply:
Unfortunately not for that one. That's because the type that I need to use is called `PowerShell` and the namespace is called `Microsoft.DotNet.Interactive.PowerShell` so they conflict.